### DISCUSSION TITLE
What is the link to get the picture?

### DISCUSSION THREAD

**Discussion by AButsai** [deta/cloud-docs#619] [Help]
Hello. 
Please tell me how to get a link to an image in the img HTML tag. 
And is there such a possibility?

For example: <img href*="https://drive.deta.sh/v1/..." alt="img" />

Thanks a lot for your help!!!

**Comment by jnsougata** [comment]
It's possible if the image is < 5MB. you can just deliver the image from a micro which opens your drive and brings the image from drive

**Reply by AButsai** [reply]
I figured out how to do this) and then I didn’t figure out how to get the link to the picture that is stored on the deta-drive). I need a link so that I can insert it into the tag <img href*=" here I need a link to the picture that lies on the deta-drive" />

**Reply by AButsai** [reply]
I need this for a website interface

**Reply by jnsougata** [reply]
add an endpoint to your micro like `https://something.deta.dev/images/example.png`
in server side you will fetch the image `example.png` from drive and send as a response

then you will be able to do `<img href="https://something.deta.dev/images/example.png" alt="img" />`

**Reply by AButsai** [reply]
Thanks for the advice! Now I'm faced with the task of how to make a picture from the buffer and send it to a website)

But your hint gave me a direction where to dribble further. Thanks!!!

**Reply by jnsougata** [reply]
> Thanks for the advice! Now I'm faced with the task of how to make a picture from the buffer and send it to a website)
> 
> But your hint gave me a direction where to dribble further. Thanks!!!

Might be helpful: 
`Python` https://docs.deta.sh/docs/drive/py_tutorial
`Node` https://docs.deta.sh/docs/drive/node_tutorial
